extensions/Lily/CommentBlocks.js

```diff
@@ -4,88 +4,105 @@
             isEdgeActivated: false,
             arguments: {
               COMMENT: {
                 type: Scratch.ArgumentType.STRING,
-                defaultValue: "comment",
-              },
-            },
+                defaultValue: 'comment'
+              }
+            }
           },
           {
-            opcode: "commentCommand",
+            opcode: 'commentCommand',
             blockType: Scratch.BlockType.COMMAND,
-            text: "// [COMMENT]",
+            text: '// [COMMENT]',
             arguments: {
               COMMENT: {
                 type: Scratch.ArgumentType.STRING,
-                defaultValue: "comment",
-              },
-            },
+                defaultValue: 'comment'
+              }
+            }
+          },
+          {
+            opcode: 'commentC',
+            blockType: Scratch.BlockType.CONDITIONAL,
+            text: '// [COMMENT]',
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'comment'
+              }
+            }
           },
           {
-            opcode: "commentReporter",
+            opcode: 'commentReporter',
             blockType: Scratch.BlockType.REPORTER,
-            text: "[INPUT] // [COMMENT]",
+            text: '[INPUT] // [COMMENT]',
             arguments: {
               COMMENT: {
                 type: Scratch.ArgumentType.STRING,
-                defaultValue: "comment",
+                defaultValue: 'comment'
               },
               INPUT: {
                 type: Scratch.ArgumentType.STRING,
-                defaultValue: "",
-              },
-            },
+                defaultValue: ''
+              }
+            }
           },
           {
-            opcode: "commentBoolean",
+            opcode: 'commentBoolean',
             blockType: Scratch.BlockType.BOOLEAN,
-            text: "[INPUT] // [COMMENT]",
+            text: '[INPUT] // [COMMENT]',
             arguments: {
               COMMENT: {
                 type: Scratch.ArgumentType.STRING,
-                defaultValue: "comment",
+                defaultValue: 'comment'
               },
               INPUT: {
-                type: Scratch.ArgumentType.BOOLEAN,
-              },
-            },
-          },
-        ],
+                type: Scratch.ArgumentType.BOOLEAN
+              }
+            }
+          }
+        ]
       };
     }
 
-    commentHat() {
+    commentHat () {
       // no-op
     }
 
-    commentCommand() {
+    commentCommand () {
       // no-op
     }
 
-    commentReporter(args) {
+    commentC (args, util) {
+      util.startBranch(1, false);
+    }
+
+    commentReporter (args) {
       return args.INPUT;
     }
 
-    commentBoolean(args) {
+    commentBoolean (args) {
       return args.INPUT || false;
     }
   }
   Scratch.extensions.register(new CommentBlocks());
+})(Scratch);
+  Scratch.extensions.register(new CommentBlocks());
```

Comment:
The extension is defined twice, is that intentional?

Reply:
who doesn't want twice as many non-functioning blocks?